upgrade-notes/storefront_20250821_000000.md

```diff
@@ -0,0 +1,24 @@
+#### Translation system extended with accessibility namespace support ([#TODO](https://github.com/shopsys/shopsys/pull/TODO))
+
+- storefront translation system now supports multiple namespaces using i18next-parser v9.3.0+ native namespace routing
+- **BREAKING CHANGE**: `aria*` translations have been moved from `common.json` to `accessibility.json` namespace
+- **Action required**: Update your translation usage from:
+
+    ```typescript
+    // Old usage
+    t('Go to cart page');
+
+    // New usage
+    t('accessibility->Go to cart page');
```

Comment:
I thought than we can use:

Example:
`const { t } = useTranslation(['common', 'accessibility']);`

```
t('This variant does not match the selected filter.', {ns:'accessibility'});
t('Ok'); ... /* this tries common and after that others */

```
In this approach we can control witch namespace is loading.